app/core/epoch_sealing_job.py

```
@@ -0,0 +1,195 @@
+    TsaAnchorRepository,
+    TenantRepository,
+)
+from app.infrastructure.services.tsa_service import TsaService
+
+logger = logging.getLogger(__name__)
+
+SEAL_POLL_INTERVAL_SECONDS = 30
+
+
+async def run_epoch_sealing_job(
+    http_client: Any,
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `http_client`

(ANN401)